CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ The changelog for `MessageKit`. Also see the [releases](https://github.com/Messa
 - `scrollsToBottomOnFirstLayout` property to automatically scroll to the bottom of `MessagesCollectionView` on first load.
 [#213](https://github.com/MessageKit/MessageKit/pull/213) by [@FraDeliro](https://github.com/FraDeliro).
 
+- `additionalTopContentInset` property to `MessagesColectionViewController` to allow users to account for extra subviews.
+[#218](https://github.com/MessageKit/MessageKit/pull/213) by [@SD10](https://github.com/SD10).
```

Comment:
Wrong PR number in link! Amateur hour over here

Reply:
Brutal. I tried to make the CHANGELOG entry before the PR this time

Reply:
😄

Reply:
😂 #savedACommit is this the new trend?

Otherwise looks good though!

Reply:
I'm trying to hide the mistake in the git history 😅 GO BACK TO WORK! 
I'm uninstalling SherloukLint